source/user-management.rst

```diff
@@ -0,0 +1,220 @@
+
+If you have added users manually (i.e. with the Method 1), the synchronization can be run manually by executing ``kubectl exec -it cluster1-proxysql-0 -- proxysql-admin --config-file=/etc/proxysql-admin.cnf --syncusers``, or you can wait until it gets run during cluster changes (Pod restart).
+
+Propagation Delays and Other Caveats
+------------------------------------
+
+When the ``proxysql-admin --syncusers`` is ran, it deletes any users which no longer exist in MySQL, so it's imperative that it gets run after users are successfully added. As such, there is a short delay during the user creation and grants process, as we run all queries first before executing syncusers.  This delay can be up to a few seconds.
+
+Additionally, if you are not making use of the automated processes for managing users, you are also responsible for manually syncing users to ProxySQL.
+
+If you have manually added some users and some have been added afterward using the automated method, syncusers will cause both sets of users to sync to ProxySQL, and the Operator will not interact with or otherwise harm the users you created manually.
+
+If you do the automated piece after the manual piece, syncusers gets run automatically.
+
+
+Administration / System Users Management
+========================================
+
+Users Required by The Operator
+------------------------------
+
+In order to automate the deployment and management of Percona XtraDB Cluster and ProxySQL, the Operator requires system-level PXC users.  The minimal set of users is ``root``, ``proxyadmin``, ``xtrabackup``, ``clustercheck``, and ``monitor``.
+
+The purposes are relatively self-evident from the names, but a detailed table can be found :ref:`in a dedicated section<users.system-users>` which describes each user and what they are utilized for with links to related documentation.
+
+YAML Object Format
+------------------
+
+The default name of the Secrets object for these users is ``my-cluster-secrets`` and can be set in the CR for your cluster in ``spec.secretName`` to something different.  When you create the object, it should match the following simple format.
+
+.. code:: yaml
+
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: my-cluster-secrets
+   type: Opaque
+   data:
+     root: cm9vdF9wYXNzd29yZA==
+     xtrabackup: YmFja3VwX3Bhc3N3b3Jk
+     monitor: bW9uaXRvcg==
+     clustercheck: Y2x1c3RlcmNoZWNrcGFzc3dvcmQ=
+     proxyadmin: YWRtaW5fcGFzc3dvcmQ=
+     pmmserver: c3VwYXxefHBheno=
+
+The example above matches what is shipped in ``deploy/secrets.yaml`` which contains default passwords. You should NOT use these in production, but they are present to assist in automated testing or simple use in a development environment.
+
+As you can see, because we use the ``data`` type in the Secrets object, all values for each key/value pair must be encoded in base64.  To do this you can simply run ``echo "password" | base64`` in your local shell to get valid values.
```

Comment:
really big mistake in fact :(
```suggestion
As you can see, because we use the ``data`` type in the Secrets object, all values for each key/value pair must be encoded in base64.  To do this you can simply run ``echo -n "password" | base64`` in your local shell to get valid values.
```